src/containers/CheckoutPage/CheckoutPageWithPayment.js

```diff
@@ -87,6 +89,7 @@ const getOrderParams = (pageData, shippingDetails, optionalPaymentParams, config
     listingId: pageData?.listing?.id,
     ...deliveryMethodMaybe,
     ...quantityMaybe,
+    ...seatsMaybe,
```

Comment:
This might give a bit false impression that quantity & seats could be used together.
Perhaps there could be a comment that it's **_quantity_** or **_seats x units_** (where units is extracted from bookingDatesMaybe) or only **_bookingDatesMaybe_** (in which case seats = 1 is assumed).